src/index.ts

```diff
@@ -256,6 +256,7 @@ export * as ipfs from './ipfs'
 export * as keystore from './keystore'
 export * as machinery from './common'
 export * as crypto from './crypto'
+export * as cbor from 'cborg'
```

Comment:
I'm not happy about exposing this, but I need this to test the browser build in `encoding.test.ts`